pkg/metrics/metrics.scss

```diff
@@ -184,18 +184,14 @@
   }
 
   // Don't wrap network in / out data
-  td[data-label="In"],
-  td[data-label="Out"] {
-    white-space: nowrap;
-  }
-
   // Shrink network cards and data a little more
   // (as space is costly and they use a lot)
   // FIXME: Let's fix the content (split off units) and drop this when we can
-  td[data-label="In"],
-  td[data-label="Out"],
-  td[data-label="Interface"] {
-    font-size: var(--pf-global--FontSize--xs);
```

Comment:
I guess this should have been a separate commit. It is fine, just for the next time

Reply:
I found and fixed a few others of these @ https://github.com/cockpit-project/cockpit/pull/17721. (I purposefully skipped fixing these there, so the PRs should not conflict.)